vnext/Microsoft.ReactNative/Views/DevMenu.cpp

```diff
@@ -106,6 +114,27 @@ void DevMenuManager::CreateAndShowUI() noexcept {
       Mso::React::ReactOptions::UseFastRefresh(m_context->Properties()) ? L"Disable Fast Refresh"
                                                                         : L"Enable Fast Refresh");
 
+#if defined(USE_HERMES)
+  devMenu.SamplingProfilerText().Text(
+      !Microsoft::ReactNative::HermesSamplingProfiler::IsStarted() ? L"Start Hermes sampling profiler"
+                                                                   : L"Stop Hermes sampling profiler");
+  std::ostringstream os;
+  if (Microsoft::ReactNative::HermesSamplingProfiler::IsStarted()) {
+    os << "Sampling profiler running..";
+  } else {
+    os << "Click to start.";
+  }
+
+  auto lastTraceFilePath = Microsoft::ReactNative::HermesSamplingProfiler::GetLastTraceFilePath();
+  if (!lastTraceFilePath.empty()) {
+    os << std::endl
+       << "Samples from last invocation are stored at " << lastTraceFilePath.c_str() << "  (path copied to clipboard).";
```

Comment:
re copy to clipboard: This reads as if it automatically at some point updates my clipboard. 
Since clipboard is a single value (Unless you remember the Offce ClipBoard tool with a stack :)) I would consider this destructive behavior and I would be upset if I lost something from my clipboard unexpectedly...

Reply:
@dannyvv you can use Windows' clipboard history (Win+V) :)
I think this feature is fine as is provided there's indication that this copies to the clipboard (e.g. a clipboard icon)

Reply:
I didn't have the clipboard history enabled when i started this ! I've now updated the message to make it explicit that stopping the profiler will update the clipboard ..